components/renku_data_services/notebooks/blueprints.py

```diff
@@ -413,13 +417,7 @@ async def _handler(
                     extraContainers=extra_containers,
                     initContainers=extra_init_containers,
                     extraVolumes=extra_volumes,
-                    culling=Culling(
-                        maxAge=f"{self.nb_config.sessions.culling.registered.max_age_seconds}s",
-                        maxFailedDuration=f"{self.nb_config.sessions.culling.registered.failed_seconds}s",
-                        maxHibernatedDuration=f"{self.nb_config.sessions.culling.registered.hibernated_seconds}s",
-                        maxIdleDuration=f"{self.nb_config.sessions.culling.registered.idle_seconds}s",
-                        maxStartingDuration=f"{self.nb_config.sessions.culling.registered.pending_seconds}s",
-                    ),
+                    culling=get_culling(resource_pool, self.nb_config),
```

Comment:
is it possible to update this when a session is resumed? In case the pool policy changes, hibernated sessions that restart will keep the old policy but it would be nice if the current one could be applied.

Reply:
if I'm reading this correctly, the resource pool is loaded from db each time and so should have up to date values, and the `get_culling` gives the values from the pool highest priority, so this should already do what you suggested?

Reply:
And when the policy changes then a session that is hibernated and then resumed will use the new culling policy.